lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2191,6 +2191,8 @@ TFGraphFunctionLowering::visitGraphOperationInst(GraphOperationInst *inst) {
     case SILTensorOpInfo::OperandClass::Array: // Handled as 'normal'
     case SILTensorOpInfo::OperandClass::ArrayElement:
```

Comment:
Randomly, I think ArrayElement and some of the other SILTensorOpInfo::OperandClass members can be removed now.  Maybe @mhong can tackle this at some point?

Reply:
Yes I can follow up on this.